scripts/audit_frontend_licenses.py

```diff
@@ -131,15 +131,6 @@
         "http://colorbrewer2.org/",
         "Cynthia Brewer",
     ),
-    (
```

Comment:
Drive by: removed because added a license to streamlit app but forgot to remove this.